utils/rpms/packaging/get_release_branch

```diff
@@ -8,17 +8,18 @@
 # base branches can be master, release/2.4+, release/3+
 # or optionally branches passed into $1
 set -eu -o pipefail
-IFS=' ' read -r -a add_bases <<< "${1:-}"
 origin="${ORIGIN:-origin}"
-all_bases=()
+builtin_bases=()
 while IFS= read -r base; do
-    all_bases+=("$base")
+    builtin_bases+=("$base")
 done < <(echo "master"
          git branch -r | sed -ne "/^  $origin\\/release\\/\(2.[4-9]\|[3-9]\)/s/^  $origin\\///p")
+all_bases=("${builtin_bases[@]}" "$@")
```

Comment:
FYI this will loop over a lot more branches now, which is slower. I'm working on making this run in parallel in #16101